### PR TITLE
Updates to front READMEs

### PR DESCRIPTION
**Pull request by gdcutting**
Filled out front README, remove GCP portion of Getting Started

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,26 @@
 
 Welcome to the public preview repository for Data Engineering Bootcamp. This repository contains introductory material and the full 10-chapter curriculum for the 17-week data engineering course, along with the datasets and code examples that we will be using to teach you how to build data pipelines.
```
Comment (xdatanomad) [inline README.md]:
```suggestion
Welcome to the intro repository for Data Engineering Bootcamp. This repository contains the first 2 chapters of our larger 10-chapter data engineering curriculum which is taught in a 17-week bootcamp. This public preview is limited to the Getting Started Guide and first two chapters of material.
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,26 @@
...
 
-If you are new to the course and need to get oriented, take a look at the [Getting Started Guide](/getting-started/) first. See the [Course Overview](deb/course-overview.md) to get a high-level view of the curriculum material and schedule.
+Take a look at the [Getting Started Guide](/getting-started/) for information on how to set up your development environment to follow along.
```
Comment (xdatanomad) [inline README.md]:
```suggestion

## Getting Started

First, take a look at the [Getting Started Guide](/getting-started/)  to set up your development environment.

This information will have you install all the necessary software to start this course.

```

File: README.md [hunk 3]
```diff
@@ -2,7 +2,26 @@
...
 
-The day-to-day instruction for the course will follow this repository material very closely. Please take at least a few minutes to [explore the /deb/ folder](/deb/) and begin to get acquainted with the structure of the course. This public preview is limited to the Getting Started Guide and first two chapters of material.
+The day-to-day instruction for the course will follow this repository material very closely. This public preview is limited to the Getting Started Guide and first two chapters of material.
```
Comment (xdatanomad) [inline README.md]:
```suggestion
```

Comment (xdatanomad) [reply README.md]: Removing this section. The information is already included above.

File: README.md [hunk 4]
```diff
@@ -2,7 +2,26 @@
...
+1. Spark
+1. Serverless architecture
+1. Streaming (Pub/Sub + Spark)
+1. Data Visualization
+
+Each chapter is contained in its own folder, which contains an overview, episode links, and code and data resources. Click on the chapter titles above to explore the chapter structure of the course. 
+
+There are ten chapters of material, each divided into daily episodes. Most of the chapters are covered in a week, but some of the bigger topics get two weeks per chapter. There are daily exercises in the form of challenge questions that ask you to do programming tasks with the skills from that day's material. Every week finishes with a graded project which ties together all of the week's material and builds on skills learned earlier in the course.
+
+The course finishes with a two-week, hands-on final project that incorporates that challenges students to use all the skills they have learned to build a real-world data pipeline.
```
Comment (xdatanomad) [inline README.md]:
```suggestion
```

Comment (xdatanomad) [reply README.md]: Removing this. It's irrelevant there.

File: getting-started/README.md [hunk 5]
```diff
@@ -146,179 +146,4 @@ Navigate to the project folder containing your local copy of the git repository
...
-
-Once your virtualenv is activated, you can can use pip to install the project dependencies:
-```
-pip install -r requirements.txt
-```
-
-VS Code has a built in Source Control interface for git. To stage and create commits, navigate to the Source Control tab by clicking on the Source Control icon, and add the files you would like to stage and commit.To change branches or create a new branch, you can click on the Git Branch icon at thebottom right of the VS code window.
-
-VS Code has a huge number of official and community extensions, so you can customize your coding environment to your heart's content.
+<br>
```
Comment (xdatanomad) [inline getting-started/README.md]: @gdcutting remove the Google Cloud setup from the list all the way on the top. It's under "Setting up your development environment". AND make that list links to the sections below

**Review by xdatanomad — CHANGES_REQUESTED** [review]

File: getting-started/README.md (hunk 5 above)
Comment (gdcutting) [reply getting-started/README.md]: I see what you mean. Took GCP out of the numbered list and then made anchor links to the sections below

**Pull request merged**